CONTRIBUTING.md

```diff
@@ -30,20 +39,34 @@ First of all, thank you for contributing to Meilisearch! The goal of this docume
 
 ## Development Workflow
 
+### Requirements
+
+- [Composer 2.x](https://getcomposer.org/download/)
+
 ### Setup
 
+Clone the project then:
+
 ```bash
+cd meilisearch-php
 composer install
 ```
 
+If you don't already have a Meilisearch server instance running, execute the following commands:
+
+NB: Will create a bunch of files and folders, better to use a dedicated folder outside of this project.
+
+```bash
+curl -L https://install.meilisearch.com | sh # download Meilisearch
+./meilisearch --master-key=masterKey --no-analytics # run Meilisearch
+```
+
```

Comment:
This command belong to the Test and Linter section because you don't need it to setup the project you will need it if you want launch the tests

Reply:
Ok, agreed.